src/sidebar/services/groups.js

```diff
@@ -199,12 +200,16 @@ function groups(
         // particular group as well since it may not be in the results returned
         // by group.list or profile.groups.
         if (directLinkedGroup) {
+          // Clear any previously set state.
+          store.clearDirectLinkedGroupFetchFailed();
           const selectedGroupApi = api.group
             .read({
               id: directLinkedGroup,
               expand: params.expand,
             })
             .catch(() => {
+              // Set the state of directLinkedGroupFetchFailed to true.
```

Comment:
This comment effectively just states what the code itself says, so I would just omit it.